corehq/apps/sms/static/sms/js/settings.js

```diff
@@ -9,185 +9,196 @@ hqDefine("sms/js/settings", [
     initialPageData,
     select2Handler
 ) {
+    $(function () {
```

Comment:
Why did the definitions of `dayTimeWindow` and `settingsViewModel` move into the document ready handler? It isn't harmful, I'm just curious if there was a reason.

Reply:
@orangejenny Don't know why but if these definitions weren't in the document ready then they are not visible when we try to execute the constructor.

Reply:
I don't follow - I'm thinking of a setup like this, where the models are outside the handler, but they'd still be accessible inside the handler:
```function dayTimeWindow (...) {
   ...
}

function settingsViewModel(...) {
   ...
}

$(function() {
   var settingsModel = settingsViewModel(...);
   ...
});```

Reply:
But in any case, the code is fine either way, so I'm going to go ahead and merge this PR.